packages/components/src/components/hds/form/field-group/index.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+import type { ComponentLike } from '@glint/template';
+
+import type { HdsLayoutFlexItemSignature } from '../../layout/flex/item.ts';
```

Comment:
I know this probably isn't the right way to do this. Should I just copy the type instead for this new component?

Reply:
No, that's the correct way. Since you're using the component, you consume its type too, so they remain in sync if one day we make a change to the flex component

Reply:
Ok thanks. Using a relative path like that just looked wrong to me.